go/libraries/doltcore/sqle/dolt_log_table_function.go

```diff
@@ -512,23 +536,27 @@ func expressionToString(ctx *sql.Context, expr sql.Expression) (string, error) {
 	return valStr, nil
 }
 
-// Gets revisionName and/or excludingRevisionName from sql expression
-func getRevisionsFromExpr(ctx *sql.Context, expr sql.Expression, canDot bool) (string, string, error) {
+// Gets revisionName and/or secondRevisionName from sql expression
+func getRevisionsFromExpr(ctx *sql.Context, expr sql.Expression, canDot bool) (string, string, bool, error) {
```

Comment:
(minor) For Go to associate the comment with the function, it needs the function name as the first word. 

```suggestion
// getRevisionsFromExpr returns the revisionName and/or secondRevisionName from sql expression
func getRevisionsFromExpr(ctx *sql.Context, expr sql.Expression, canDot bool) (string, string, bool, error) {
```

Reply:
Also, would be helpful for future code-readers to document what that boolean return parameter is. That would be difficult for someone to figure out without having to jump in and read the function code.